Adafruit_SSD1306.cpp

```diff
@@ -506,7 +513,7 @@ void Adafruit_SSD1306::display(void) {
   else
   {
     // save I2C bitrate
-#ifndef __SAM3X8E__
+#if !defined(__SAM3X8E__) && !defined(__SAMD21G18A__)
     uint8_t twbrbackup = TWBR;
     TWBR = 12; // upgrade to 400KHz!
 #endif
```

Comment:
The TWBR related changes in this PR should be not needed anymore in the current master branch.